.env.sample

```diff
@@ -3,5 +3,10 @@ POSTGRES_PASSWORD=
 POSTGRES_DB=
 DBHOST=postgres
 DBPORT=5432
+
 path_model=https://github.com/SAP/credential-digger/releases/download/PM-v1.0.1/path_model-1.0.1.tar.gz
 snippet_model=https://github.com/SAP/credential-digger/releases/download/SM-v1.0.0/snippet_model-1.0.0.tar.gz
+
+# Enable HTTPS by setting the path of both the certificate and the private key as shown below
+SSL_certificate=/credential-digger-ui/$PATH_to_certificate # i.e /credential-digger-ui/cert.pem
```

Comment:
`/credential-digger-ui/cert.pem` are not actual folders of user's file system. This path is the path that will be created in the docker container, but a standard user doesn't have visibility over this. So, I suggest to leave the path here as the path where to find the certificate (and key) on the local file system, and then they will be put in a suitable folder in the docker container.
In practice, as it happens with the `rules.yaml` file for the rules